cmd/kubeadm/app/cmd/init_test.go

```diff
@@ -45,7 +51,7 @@ nodeRegistration:
 apiVersion: %[1]s
 kind: ClusterConfiguration
 controlPlaneEndpoint: "3.4.5.6"
-`, kubeadmapiv1.SchemeGroupVersion.String())
+`, v1beta3.SchemeGroupVersion.String())
```

Comment:
kubeadmspiv1 is better here as import name, so that we have smaller diff when the api becomes v4